.github/workflows/formioLibUpdate.js.yml

```diff
@@ -0,0 +1,54 @@
+name: Update Libs
+on:
+  schedule:
+    # * is a special character in YAMLso you have to quote this string
+    - cron:  '30 5 * * *'
+jobs:
+  updateChangelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: "main"
+      # run script in prem, extract to lib, update versions.txt on each file
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: 'npm'
+          cache-dependency-path: perm/package-lock.json
+          registry-url: 'https://registry.npmjs.org'
+      - name: List versions
+        working-directory: ./perm
+        run: | # List versions
+          npm view formiojs versions
+          npm view @formio/premium versions
+        env:
```

Comment:
`env` can be defined at a higher level instead of being repeated.

Reply:
umm, i did not 'alter' those workflow's which failed. What was run was 
https://github.com/qld-gov-au/formio/actions/runs/2193583748/workflow which had no registry changes.

Reply:
Sure, the `env` suggestion is unrelated to the build failure. It's just a suggestion. I'm unsure where the authentication failure is coming from.